src/styles/blocks/card.css

```diff
@@ -0,0 +1,94 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+}
+
+.card__image {
+  padding: 0 3px;
+  max-height: 134px;
+}
+
+.card__product-name {
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+  line-height: 18px;
+
+  padding: 40px 0 4px;
```

Comment:
For some elements you are using paddings to set distance between them, for other you used margins. Let's be consistent, since you don't need to increase clickable zone of headings or image - use margins for both